preprocessor.h

```diff
@@ -119,7 +141,7 @@ class Preprocessor
      * preprocessor. */
     bool is_included()
     {
-        if (stack_.empty() || stack_.back().condition)
+        if (stack_.empty() || stack_.back().condition == True)
```

Comment:
Do we need a new condition type 'Skip' or can we just do:
```c
bool is_included()
{
    for (auto& s : stack_)
        if (!s.condition)
            return false;
    return true;
}
```

Reply:
Sure, that also works.

Reply:
updated.